screens/shopsearchscreen.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
```

Comment:
Remove this file - make sure that your **only** checkout from the master branch. Since you checked out from your `shopsearchscreen` you included that branch's code into this branch which duplicates the amount of code.